src/module/item/item-sheet.js

```diff
@@ -293,4 +340,50 @@ export class ItemArchmageSheet extends ItemSheet {
       }
     }).render(true);
   }
+
+  /* ------------------------------------------------------------------------ */
+  /*  Handle effects -------------------------------------------------------- */
+  /* ------------------------------------------------------------------------ */
+  _onManageEffect(event) {
+    let target = event.currentTarget;
+    let dataset = target.dataset;
+    const effect = dataset.itemId ? this.document.effects.get(dataset.itemId) : null;
+
+    switch (dataset.action) {
+      case 'create':
+        return this.document.createEmbeddedDocuments('ActiveEffect', [{
+          label: game.i18n.localize("ARCHMAGE.EFFECT.AE.new"),
+          icon: 'icons/svg/aura.svg',
+          origin: this.document.uuid,
```

Comment:
This will likely need to be adjusted to attempt to reference the parent actor instead of the item itself when present.